Dockerfile.devkit

```diff
@@ -72,40 +72,46 @@ RUN mkdir -p /usr/share/ansible/collections \
     -p /usr/share/ansible/collections
 
 # hadolint ignore=DL4006
-RUN  curl -Lf https://github.com/mesosphere/mindthegap/releases/download/v"${MINDTHEGAP_VERSION}"/mindthegap_v"${MINDTHEGAP_VERSION}"_linux_amd64.tar.gz |tar xzf - -C /usr/local/bin
+RUN curl -Lf https://github.com/mesosphere/mindthegap/releases/download/v"${MINDTHEGAP_VERSION}"/mindthegap_v"${MINDTHEGAP_VERSION}"_linux_amd64.tar.gz | tar xzf - -C /usr/local/bin
+
+# Goss is used for testing the image after it is built.
+# The binary will be copied to the remote host and only linux-amd64 is supported.
 ARG GOSS_VERSION=v0.3.23
-RUN curl -L "https://github.com/goss-org/goss/releases/download/${GOSS_VERSION}/goss-linux-amd64" -o /usr/local/bin/goss-amd64
-RUN chmod +rx /usr/local/bin/goss-amd64
-ARG BUILDARCH
```

Comment:
We were inconsistently using `BUILDARCH` here, this goss binary is used on the remove machine where only linux-amd64 is supported.
The Dockerfile will always look for the amd64 version
https://github.com/mesosphere/konvoy-image-builder/blob/187d011f8f9871fb0738821f2807c664e3653961/Dockerfile#L32-L33

Reply:
thanks for fixing this.